### PR TITLE
Fixed flaky integration tests

### PR DESCRIPTION
**Pull request by andrewbess**
### Description (*)

This pull request provides the fix for flaky integration tests.

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/591

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Comment by andrewbess** [comment]
@magento run Database Compare, Semantic Version Checker, WebAPI Tests, Unit Tests, Static Tests, Magento Health Index, Functional Tests CE, Functional Tests EE, Functional Tests B2B, Sample Data Tests EE, Sample Data Tests CE, Sample Data Tests B2B

File: dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/AbstractProductExportImportTestCase.php [hunk 1]
```diff
@@ -345,7 +345,7 @@ protected function executeImportReplaceTest(
                 'name' => '_itemsPerPage'
             ]);
             $itemsPerPageProperty->setAccessible(true);
-            $itemsPerPageProperty->setValue($exportProduct, 1);
+            $itemsPerPageProperty->setValue($exportProduct, 2);
```
Comment (sivaschenko) [inline dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/AbstractProductExportImportTestCase.php]: @andrewbess can you please add a description to the pull request. Why 2?

**Comment by andrewbess** [comment]
@magento run Integration Tests

**Comment by fascinosum** [comment]
@magento run Integration Tests

**Comment by andrewbess** [comment]
@magento run Database Compare, Semantic Version Checker, WebAPI Tests, Unit Tests, Static Tests, Magento Health Index, Functional Tests CE, Functional Tests EE, Functional Tests B2B, Sample Data Tests EE, Sample Data Tests CE, Sample Data Tests B2B

**Comment by andrewbess** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Review by eduard13 — APPROVED** [review]

**Pull request merged**